source/agora/utils/HumanReadable.d

```diff
@@ -0,0 +1,284 @@
+/*******************************************************************************
```

Comment:
`reduce` => `improves` in the commit message imo. Unless you also means it makes it more compact.

Reply:
Yes, the point is not so much to have something "sensible", but something that doesn't take a whole screen.